modules/serverless-rn-1-7-1.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-7-1_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.7.1
+
+* {ServerlessProductName} now uses Knative Serving 0.13.3.
+* {ServerlessProductName} now uses Knative Serving Operator 0.13.3.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.13.2.
+* {ServerlessProductName} uses Knative Eventing 0.13.3.
```

Comment:
let's say this is 0.13.0 - thanks

Reply:
Let's also mention Knative Eventing Operator 0.13.3 as well. The version can be different from the core Eventing components, no problem.